angular/wolox-books/src/app/screens/register/register.component.html

```diff
@@ -23,6 +23,6 @@
     <span class="alert" *ngIf="!registerForm.controls['password'].valid && registerForm.controls['password'].touched">{{fieldRequired}}</span>
     <button class="btn" type="submit">Sign Up</button>
     <hr class="separator">
-    <button class="btn btn-outline" type="button">Login</button>
+    <button class="btn btn-outline" type="button" (click)="goToLogin()">Login</button>
```

Comment:
you can use `routerLink` instead of a function since it's only action is the change of route. also when changing routes use anchor tags and when doing actions in the same view use buttons (buttons might also be used if an action is done in the same view and after that action a redirection is done)

https://angular.io/api/router/RouterLink